src/components/connections/SummaryHeader.tsx

```diff
@@ -48,18 +59,33 @@ const SummaryHeader = ({
     }
   };
 
-  const generateCSV = () => {
-    const blob = generateJourneyCsvService(
-      selectedConnectionSummary?.['connections'],
-      selectedConnectionSummary?.endOrgan?.name ?? '',
-      filters,
+  pdfMake.fonts = {
```

Comment:
nitpick: I would move this to the pdf service

Reply:
I moved part of the docDefinition to the pdf service, but I believe the `fonts` and `window.pdfMake.vfs` are mainly component-level definitions. I would say it's better to keep them within the component.